discord/interactions.py

```diff
@@ -306,10 +306,28 @@ async def original_message(self) -> InteractionMessage:
         )
         state = _InteractionMessageState(self, self._state)
         message = InteractionMessage(state=state, channel=channel, data=data)  # type: ignore
-        self._original_message = message
+        self._original_response = message
         return message
 
-    async def edit_original_message(
+    @utils.deprecated("Interaction.original_response", "2.1")
+    async def original_message(self):
+        """An alias for :meth:`original_response`.
+        
+        Raises
+        -------
+        HTTPException
+            Fetching the original response message failed.
+        ClientException
+            The channel for the message could not be resolved.
+
+        Returns
+        --------
+        InteractionMessage
+            The original interaction response message.
+        """
+        return self.original_response()
```

Comment:
This being async but not calling the asynchronous Interaction.original_response() with await has caused a breaking change in which caused the result of `await Interaction.original_message()` to be a coroutine instead of the response. 👎🏻

Reply:
This was fixed in #1681.

Reply:
Any idea why the fix is being sat on being released?

Reply:
You can use the development version of the library or wait for the next release.
You could also use `Interaction.original_response` instead.